demo/simple.html

```diff
@@ -9,7 +9,7 @@
         <script src="../jquery.orgchart.js"></script>
         <script>
         $(function() {
-            $("#organisation").orgChart({container: $("#main")});
+            $("#organisation").orgChart({container: $("#main") , direction:'ltr'});
```

Comment:
I should not need to specify direction LTR here, it's supposed to be the default.